src/frontends/pytorch/src/transforms/u4_block_repack.hpp

```diff
@@ -18,6 +18,12 @@ class U4BlockRepack : public ov::pass::MatcherPass {
     U4BlockRepack(bool is_symmetrical = false);
 };
 
+class U4ConvertReshape : public ov::pass::MatcherPass {
```

Comment:
can you please add description to this transformation? What do we expect from it?

Reply:
Thanks for the comment. I updated PR description.